src/main/java/uk/gov/justice/digital/hmpps/cmd/api/service/DryRunNotificationService.kt

```diff
@@ -76,9 +76,13 @@ class DryRunNotificationService(
         .sortedWith(compareBy<CsrModifiedDetailDto> { it.quantumId }.thenByDescending { it.shiftModified })
         .distinctBy {
           CsrModifiedDetailDto(
-            null, it.quantumId, null, it.shiftType,
-            it.detailStart.truncatedTo(ChronoUnit.DAYS), it.detailEnd.truncatedTo(ChronoUnit.DAYS),
-            null, it.actionType,
+            quantumId = it.quantumId,
+            shiftModified = null,
```

Comment:
surprised you need to pass in `null` here, would maybe be better to set defaults in `CsrModifiedDetailDto` for those fields to be `null` and then not include them in the parameters